lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +229,89 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        mapping_dir = Path(__file__).parent / 'data'
+        if not mapping_dir.exists():
+            mapping_dir.mkdir(parents=True)
+        mapping_file = mapping_dir / 'mystem_tags_mapping.json'
+        if not mapping_file.exists():
+            mapping_file.touch()
+        if not mapping_file.stat().st_size:
+            ud_mapping = {'POS': {'S': 'NOUN',
```

Comment:
mapping should be defined in a JSON file

Reply:
Done